scheduler/services/resource/test/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
 # For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-# This file is needed for the test_resource_mock.py file to run.
+# This file is needed for the test_ocs_resource_service.py file to run.
```

Comment:
Test are now located in the tests folder at the top.

Reply:
Thanks! I'll move it.

Reply:
Never mind... since we don't have any official tests, I just removed it.